http-server-netty/src/main/java/io/micronaut/http/server/netty/handler/accesslog/HttpAccessLogHandler.java

```diff
@@ -50,11 +52,17 @@
      */
     public static final String HTTP_ACCESS_LOGGER = "HTTP_ACCESS_LOGGER";
 
+    /**
+     * A lenient filter that allows all uri's to pass.
+     */
+    public static final Predicate<String> INCLUDE_ALL = uri -> true;
```

Comment:
Perhaps use `null` as the value for `INCLUDE_ALL` as it will add a little less overhead

Reply:
Good shout.  Done 👍